### PR TITLE
add Jupyter Notebook to overview

### PR DESCRIPTION
**Pull request by KittyYeungQ**

### CODE REVIEW

**Comment by KittyYeungQ** [comment]
taken care of by #838

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/overview/what-is-qsharp-and-qdk.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/24d87e18f6ed7000c720b18a24d079bbe4c889a7/articles/overview/what-is-qsharp-and-qdk.md) | 88 | 60 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cf16a1fb-49b1-42d3-992e-b7ff7414a5fe_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by KittyYeungQ** [comment]
#838 was closed

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/overview/what-is-qsharp-and-qdk.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/24d87e18f6ed7000c720b18a24d079bbe4c889a7/articles/overview/what-is-qsharp-and-qdk.md) | 88 | 60 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1cab9743-721a-4670-b24e-21fccb955b8d_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/overview/what-is-qsharp-and-qdk.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ Everything you need to write and run Q# programs, including the Q# compiler, the
 
 - Running [Q# from the command line](xref:microsoft.quantum.install.standalone) is the quickest way to get started.
 
+- Run standalone Q# programs using [Jupyter Notebooks with IQ#](xref:microsoft.quantum.install.jupyter), an extension to the .NET Core SDK used by Jupyter and Python to provide the core functionality for compiling and simulating Q# operations.
```
Comment (KittyYeungQ) [inline articles/overview/what-is-qsharp-and-qdk.md:57]:
```suggestion
- Run standalone [Jupyter Notebooks with IQ#](xref:microsoft.quantum.install.jupyter), a Jupyter extension for compiling, simulating, and visualizing Q# programs.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/overview/what-is-qsharp-and-qdk.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/27c8e43b281358d7710d2e4bfce86283fc7bc3bd/articles/overview/what-is-qsharp-and-qdk.md) | 88 | 60 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/72cda98d-ffe0-468c-b6a6-6c4f2357de1e_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**